### PR TITLE
Fix standalone build

### PR DESCRIPTION
**Pull request by frantisekz**
Fixes https://github.com/intel/opencl-clang/issues/406 
Fixes https://github.com/intel/opencl-clang/issues/374

@haonanya wdyt? I am not proficient with cmake/compiled languages, but it seems right (and solves the mentioned issues).

### CODE REVIEW

File: cl_headers/CMakeLists.txt [hunk 1]
```diff
@@ -1,5 +1,9 @@
 set(CL_HEADERS_LIB cl_headers)
-set(CLANG_COMMAND $<TARGET_FILE:clang> )
+if(USE_PREBUILT_LLVM)
+    find_program(CLANG_COMMAND clang PATHS ${LLVM_TOOLS_BINARY_DIR} NO_DEFAULT_PATH)
+else()
+    set(CLANG_COMMAND $<TARGET_FILE:clang> )
```
Comment (hewj03) [inline cl_headers/CMakeLists.txt:5]:
```suggestion
    set(CLANG_COMMAND $<TARGET_FILE:clang>)
```

Comment (frantisekz) [reply cl_headers/CMakeLists.txt]: Addressed.

**Review by hewj03 — APPROVED** [review]
LGTM

**Comment by hewj03** [comment]
cherry-pick to ocl-open-160 branch: https://github.com/intel/opencl-clang/pull/437

**Pull request merged**